ethsigner/signer/multikey/src/main/java/tech/pegasys/ethsigner/signer/multikey/MultiKeySubCommand.java

```diff
@@ -35,7 +35,9 @@
 @Command(
     name = MultiKeySubCommand.COMMAND_NAME,
     description =
-        "This command allows metadata file that matches the sender address to specify signing credentials from one of several types",
+        "This command allows Ethsigner to access multiple keys, and use the appropriate key "
```

Comment:
Somehow think we should mention that allow you use keys from any of the signers.

Reply:
Thinking more that we should just point to the docs to explain it all - I don't want to document the full TOML format here :/ (i.e. how far do we try and go here?)